conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/WebApplicationExceptionMapper.java

```diff
@@ -31,7 +31,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * @deprecated Services should throw {@link com.palantir.remoting.api.errors.ServiceException}s instead.
+ * @deprecated Services should throw {@link com.palantir.conjure.java.api.errors.ServiceException}s instead.
  */
 @Provider
 @Deprecated
```

Comment:
separate pr for removing of the deprecated methods?

Reply:
yes